Detections/SecurityAlert/DetectPIMAlertDisablingActivity.yaml

```diff
@@ -0,0 +1,43 @@
+id: 1f3b4dfd-21ff-4ed3-8e27-afc219e05c50
+name: Detect PIM Alert Disabling activity
+description: |
+  'This query will look at AuditLogs to detect Privileged Identity Management(PIM) alerts being disabled'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AuditLogs
+queryFrequency: 1d
+queryPeriod: 1d
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Persistence
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1098
+  - T1078
+query: |
+  AuditLogs
+  | where LoggedByService =~ "PIM"
+  | where Category =~ "RoleManagement"
+  | where ActivityDisplayName has "Disable PIM Alert"
+  | extend IpAddress = case(
+    isnotempty(tostring(parse_json(tostring(InitiatedBy.user)).ipAddress)) and tostring(parse_json(tostring(InitiatedBy.user)).ipAddress) != 'null', tostring(parse_json(tostring(InitiatedBy.user)).ipAddress), 
+    isnotempty(tostring(parse_json(tostring(InitiatedBy.app)).ipAddress)) and tostring(parse_json(tostring(InitiatedBy.app)).ipAddress) != 'null', tostring(parse_json(tostring(InitiatedBy.app)).ipAddress),
+    'Not Available')
+  | extend InitiatedBy = iff(isnotempty(tostring(parse_json(tostring(InitiatedBy.user)).userPrincipalName)), 
+    tostring(parse_json(tostring(InitiatedBy.user)).userPrincipalName), tostring(parse_json(tostring(InitiatedBy.app)).displayName)), UserRoles = tostring(parse_json(tostring(InitiatedBy.user)).ipAddress)
+  | project InitiatedBy, ActivityDateTime, ActivityDisplayName, IpAddress, AADOperationType, AADTenantId
```

Comment:
We should bring thru more columns in the results for it to be more useful.  ResourceId should always be brought thru as this a primary entity for mapping in Azure.  Also, CorrelationId can be used to connect to other activity.  Lastly, it is nice to have the Identity field as it is the friendly name for the UPN.